src/pages/HackathonEventsPage/HackathonEventsPage.scss

```diff
@@ -0,0 +1,71 @@
+@use "../../styles/mixins" as *;
+@use "../../styles/variables" as *;
+@use "../../styles/typography" as *;
+
+.event-page {
+  width: 100%;
+  background: linear-gradient(
+    to bottom,
+    #c8e7fb 5%,
+    $MVP-White 20%,
+    $MVP-White 20%
+  );
+  background-repeat: no-repeat;
+  height: 100%;
+  font-family: Gilroy;
+
+  &__banner {
+    height: 25%;
+    background-color: #D9D9D9;
+
+    &__btn-navigate-back {
+      color: black;
+    }
+
+    &__header {
+      @include LandingPage-Text;
+      font-family: 'Corben';
+    }
+  }
+
+  &__btn {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    font-size: 1rem;
+    gap: 1.2rem;
+
```

Comment:
In Sass, we should generally avoid chaining multiple elements, such as 'Block__element__element', as this can lead to less maintainable code. Typically, the proper structure is 'block__element--modifier' to keep the CSS more readable and manageable.